polus-generic-to-image-collection-plugin/Dockerfile

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Rename plugin  to be more descriptive: "generic2image-collection Plugin" or something along those lines